modules/tinymce/src/core/main/ts/api/util/ImageUploader.ts

```diff
@@ -35,8 +35,8 @@ export const createUploader = (editor: Editor, uploadStatus: UploadStatus) =>
 
 /**
  * This class handles uploading images to a back-end server.
- * <br>
- * <em>Added in TinyMCE 5.7</em>
+ *
+ * @summary Added in TinyMCE 5.7.
```

Comment:
Is this correct? If so, should this be in an `<em>` block still?

Reply:
```suggestion
 * @summary <em>Added in TinyMCE 5.7.</em>
```

Reply:
Oh, hm. I removed the `em` because I put it in the summary. It seemed odd to italicise just that paragraph. Still could do it though, Idm either way.

Reply:
Talked to Tyler, made it bold.